doc/software-design-development-guidance/index.md

```diff
@@ -0,0 +1,326 @@
+# Software design development guidance
+
+!!! note
+    Work in Progress
+
+**Status**: first sketch, work in progress, request for collaboration
+
+**Date**: Updated 2025-04-01
+
+**Governance**: To Be Discovered; potentially a combo of this repo participants, DHCW CISO, NHS Wales UCB peers, etc.
+
+## Situation - Context and Problem Statement
+
+Broadly we need to design and develop many software products for patients and healthcare workers, and we want our design and development to emphasize usability, accessibility, maintainability, and similar system quality attributes.
+
+Narrowly we need to immediately design and develop the emergency department module, and we need to give the program some UI/UX guardrails:
+
+* Design system: e.g. how should a top navigation menu look and feel to the user.
+
+* Development system: e.g. how should a programmer implement the menu with JavaScript/TypeScript.
+
+* Testing system: e.g. how to test the menu for accessibility, multi-language capability, etc.
+
+## Background - Decision Drivers
```

Comment:
Thoughts on adding info from or just link to...  https://benholliday.com/2025/03/28/everything-service-patterns/ ?

Reply:
Need to resolve this before closing the review.